vllm/multimodal/utils.py

```diff
@@ -0,0 +1,71 @@
+import base64
+from io import BytesIO
+from typing import Union
+
+import requests
+from PIL import Image
+
+from vllm.config import ModelConfig
+from vllm.envs import VLLM_IMAGE_FETCH_TIMEOUT
+from vllm.multimodal.image import ImagePixelData
+from vllm.utils import make_async
+
+
+def load_image_from_base64(image: Union[bytes, str]) -> Image.Image:
+    """Load image from base64 format"""
+    return Image.open(BytesIO(base64.b64decode(image)))
+
+
+def encode_image_base64(image: Image.Image, format: str = 'JPEG') -> str:
```

Comment:
where is this used?

Reply:
This is added only for testing now, but I figured this could be a helpful util for the developer/user to encode their image so I didn't put it in the test file.